recipes/szip/all/test_package/CMakeLists.txt

```diff
@@ -1,14 +1,12 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package C)
+cmake_minimum_required(VERSION 3.8)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+project(test_package C)
 
 find_package(szip REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
 if(TARGET szip-shared)
-    target_link_libraries(${PROJECT_NAME} szip-shared)
-else()
-    target_link_libraries(${PROJECT_NAME} szip-static)
-endif()
+   target_link_libraries(${PROJECT_NAME} szip-shared)
+else ()
```

Comment:
(insert a meme about whitespace)

Reply:
heh I checked the incremental commit, but you can only see that whitespace change easily with a squashed commit or a combined diff or github Changes view.